# ALL:ALL - Se agrega -liconv a Libs en sdl2.pc.in para que funcione el enlace estatico cruzado correctamente.
--- sdl2/sdl2.pc.in
+++ sdl2/sdl2.pc.in
@@ -10,5 +10,5 @@
 Version: @SDL_VERSION@
 Requires.private: @PKGCONFIG_DEPENDS@
 Conflicts:
-Libs: -L${libdir} @SDL_RLD_FLAGS@ @SDL_LIBS@ @PKGCONFIG_LIBS_PRIV@ @SDL_STATIC_LIBS@
+Libs: -liconv -L${libdir} @SDL_RLD_FLAGS@ @SDL_LIBS@ @PKGCONFIG_LIBS_PRIV@ @SDL_STATIC_LIBS@
 Cflags: -I${includedir} -I${includedir}/SDL2 @SDL_CFLAGS@
